From 4632634c7a689970e35454e2ad70ebeccefe6385 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 2 Jun 2021 03:28:17 -0700
Subject: [PATCH 1633/1921] net: mvpp2: add driver dependency to NET_DSA

Since driver has dsa notifier callbacks

Change-Id: Icc8a7911dfe2d599c2a4126bf145b9043e7b2c85
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/55377
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/marvell/Kconfig b/drivers/net/ethernet/marvell/Kconfig
index fb942167ee54..ffad8b5ef8f4 100644
--- a/drivers/net/ethernet/marvell/Kconfig
+++ b/drivers/net/ethernet/marvell/Kconfig
@@ -84,6 +84,7 @@ config MVNETA_BM
 config MVPP2
 	tristate "Marvell Armada 375/7K/8K network interface support"
 	depends on ARCH_MVEBU || COMPILE_TEST
+	depends on NET_DSA
 	select MVMDIO
 	select PHYLINK
 	---help---
-- 
2.31.1

